packages/hydrogen/package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "build": "tsup --clean --config ../../tsup.config.ts && npm run copy-hydrogen-react",
     "copy-hydrogen-react": "node ../../scripts/copy-hydrogen-react.mjs",
-    "dev": "tsup --watch --config ../../tsup.config.ts",
+    "dev": "tsup --watch --config ../../tsup.config.ts --watch ../../node_modules/@shopify/hydrogen-react/dist/browser-prod/index.mjs",
```

Comment:
maybe `../hydrogen-react/dist/browser-prod/index.mjs` better? The node_modules one should be a symlink to this one.

Also, what's this for? Types? 🤔